primitives/src/locktime/absolute.rs

```diff
@@ -7,7 +7,8 @@
 
 use core::cmp::Ordering;
 use core::fmt;
-
```

Comment:
Same comment as elsewhere but it's nicer to read if there aren't these line removals imo.